generators/styles/index.js

```diff
@@ -37,6 +38,8 @@ module.exports = class extends Generator {
 
   writing() {
     this.log(chalk.green('Writing styles files...'));
+    // eslint-disable-next-line
+    console.log('this.props.deps', this.props.deps);
```

Comment:
Looks like a log snuck in here.

Reply:
LOL, that's the issue with those `// eslint-disable` comments!

Reply:
Removed, but of course the tests are going to fail again now. The circle of life.